packages/flutter_tools/lib/src/ios/fallback_discovery.dart

```diff
@@ -173,6 +173,10 @@ class FallbackDiscovery {
               'success',
               flutterUsage: _flutterUsage,
             ).send();
+
+            // We absolutely must dispose this vmService instance, otherwise
+            // DDS will fail to start.
+            vmService.dispose();
```

Comment:
nit: don't use "we"

Can this be tested somehow?

Reply:
I'm not sure if it's worth testing here since it's tested in the SDK, but DDS will throw an exception if another client is already connected to the VM service.

Reply:
I suppose if this were accidentally removed, the devicelab tests would already be failing.

Reply:
Yeah, the last time I tried to land this after I stopped swallowing the exception I found this issue. The iOS devicelab tests were very unhappy so it'd be hard to miss.